lib/lightstep/tracer/tracer.rb

```diff
@@ -0,0 +1,374 @@
+require 'json'
```

Comment:
(could make this comment anywhere)

IMO the paths here should be `lib/lightstep/(class).rb`... that would be more consistent with other languages.